pkgs/by-name/ya/yandex-music/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  fetchurl,
+  stdenvNoCC,
+  lib,
+  makeWrapper,
+
+  p7zip,
+  asar,
+  jq,
+  python3,
+  electron,
+  fetchFromGitHub,
+
+  electronArguments ? "",
+  trayEnabled ? true,
+}:
+let
+  version = "5.13.2";
+  src = fetchFromGitHub {
+    owner = "cucumber-sp";
+    repo = "yandex-music-linux";
+    rev = "v${version}";
+    hash = "sha256-yQX2GEulAQ02BndpwAAwGdTYtUlNXCgsCUdmXxPwtKU=";
+  };
+  version_info = builtins.fromJSON (builtins.readFile "${src}/utility/version_info.json");
+in
```

Comment:
Nit: these can be accessed through `finalAttrs` witout depending on let.

Reply:
just moved the let to `ymExe`